doc/over_react_redux_documentation.md

```diff
@@ -657,4 +658,59 @@ var store = new DevToolsStore<AppState>(
   initialState: /*Default App State Object*/,
   middleware: [overReactReduxDevToolsMiddlewareFactory(name: 'Some Custom Instance Name')],
 )
-```
\ No newline at end of file
+```
+
+### Integration With DevTools
+
+In order to display the properties of Dart based `Action`s and `State` in the DevTools they must implement a `toJson` method. 
+
+`toJson` can be manually added to the classes, or added with the help of something like the the [json_serializable](https://pub.dev/packages/json_serializable) library. 
```

Comment:
Maybe we should mention that all values need to be json encodable if a value contains objects that are not directly encodable to JSON (a value that is not a number, boolean, string, null, list or a map with string keys) we will attempt to call `toJson` on it.